examples/network/nodeStyles/images.html

```diff
@@ -68,7 +68,7 @@
       network = new vis.Network(container, data, options);
     }
   </script>
-  
+
```

Comment:
Why?

Reply:
That's just visual studio code removing trailing spaces I believe.